src/assets/mapStyles/mapStyles.js

```diff
@@ -45,7 +45,7 @@ export default {
             {
                 'id': 'background',
                 'paint': {
-                    'background-color': 'hsl(47, 26%, 88%)'
+                    'background-color': '#F5F5F5'
```

Comment:
oops didn't mean to commit this, was just playing locally with grayer than creamier